testbed/eks/lib/testbed-stack.ts

```diff
@@ -0,0 +1,69 @@
+import * as iam from '@aws-cdk/aws-iam';
+import * as eks from '@aws-cdk/aws-eks';
+import * as ec2 from '@aws-cdk/aws-ec2';
+import * as cdk from '@aws-cdk/core'
+import { FluxV2, Repository } from './addons/fluxv2';
+
+export interface TestbedProps extends cdk.StackProps {
```

Comment:
The cdk ecosystem uses both terms "props" and "options": https://github.com/cdk8s-team/cdk8s/blob/master/examples/typescript/podinfo/lib/ingress.ts#L36. I'm a bit partial to props, but happy to go either way.